C++/include/utility/string.hpp

```diff
@@ -0,0 +1,36 @@
+/*
+	Split string Function
+	----------------------
+	Breaks a given sentence into individual parts (namely words) defined
+	by a character (named delimiter) and returns all the splitted words
+	as a vector of string.
+	The function takes two arguments as input a string sentence and a
+	character delimiter (Eg: ' ', '-', 'c', '!').
+
+	Time complexity
+	---------------
+	O(n), where n is the length of the string.
+
+	Space complexity
+	----------------
+	O(n), where n is the length of the string.
+*/
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+std::vector<std::string> split_string(std::string sentence, char delimiter)
```

Comment:
Take the string parameter as a constant reference (`const std::string& str`), and name it `str` because it can be any general string and not always a sentence.